keyboards/tetris/keymaps/default/keymap.c

```diff
@@ -1,829 +1,285 @@
- *                       | HUI  | SAI  | VAI  |      |      |      |      |      |      |      |      |      |
- *                       |------+------+------+------+------+------+------+------+------+------+------+------|
- *                       | HUD  | SAD  | VAD  |      |      |    RGB_TOG  |      |      |      |      |      |
- *                       `-----------------------------------------------------------------------------------'
- */
-
-[_RGB] = LAYOUT_planck_mit(
-  _______, _______, _______,  _______, _______, RGB_TAP, _______, _______, _______, _______, _______, _______,
-  RGB_MOD, RGB_M_R, RGB_M_SW, RGB_M_K, RGB_M_X, RGB_M_G, _______, _______, _______, _______, _______, _______,
-  RGB_HUI, RGB_SAI, RGB_VAI,  _______, _______, _______, _______, _______, _______, _______, _______, _______,
-  RGB_HUD, RGB_SAD, RGB_VAD,  _______, _______,     RGB_TOG,      _______, _______, _______, _______, _______ ),
-
-/* Func
- *                       ,-----------------------------------------------------------------------------------.
- *                       |      |      |      |      |      |      |      |      |      |      |Layer0| Rest |
- *                       |------+------+------+------+------+-------------+------+------+------+------+------|
- *                       | Caps |      |      |      |      | Game |      |      |      |      |      |      |
- *                       |------+------+------+------+------+------|------+------+------+------+------+------|
- *                       |      |      |      |      |      |      | Numb | Music|      |      |      |      |
- *                       |------+------+------+------+------+------+------+------+------+------+------+------|
- *                       |      |      |      |      |      |             |      |      |      |      |      |
- *                       `-----------------------------------------------------------------------------------'
- */
-
-[_ADJUST] = LAYOUT_planck_mit(
-  _______, _______, _______, _______, _______, _______,   _______,   _______, _______, _______, TO( 0 ), RESET  ,
-  KC_CAPS, _______, _______, _______, _______, TO(_GAME), _______,   _______, _______, _______, _______, _______,
-  _______, _______, _______, _______, _______, _______,   TO(_NUMB), MU_TOG,  _______, _______, _______, _______,
-  _______, _______, _______, _______, _______,      _______,         _______, _______, _______, _______, _______ ),
-
-[_GAME] = LAYOUT_planck_mit(
-  XXXXXXX,KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    XXXXXXX,
-  XXXXXXX,KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
-  XXXXXXX,KC_LCTL, KC_A,    KC_S,    KC_D,    KC_B,    KC_N,    KC_M,    XXXXXXX, XXXXXXX, KC_UP,   TO( 0 ),
-  XXXXXXX,KC_LSFT, KC_Z,    KC_X,    KC_C,       KC_SPC,        XXXXXXX, XXXXXXX, KC_LEFT, KC_DOWN, KC_RGHT ),
+  KEYMAP(
```

Comment:
Please revert this:
```suggestion
  [_BASE] = LAYOUT_ortho_5x12(
```

As well as the rest

Reply:
This should actually be `LAYOUT_planck_mit` rather than `LAYOUT_ortho_5x12`.